src/lib/features/project/project-read-model-type.ts

```diff
@@ -0,0 +1,37 @@
+import type { ProjectMode } from '../../types';
+import type { IProjectQuery } from './project-store-type';
+
+export type ProjectForUi = {
+    id: string;
+    name: string;
+    description?: string;
+    health: number;
+    createdAt: Date;
+    mode: ProjectMode;
+    memberCount: number;
+    favorite: boolean;
+    archivedAt?: Date;
+    featureCount: number;
+    lastUsage: Date | null;
+    lastUpdated: Date | null;
```

Comment:
we can change the names of these properties! Consider them placeholders for now, but please give suggestions.

Reply:
lastReportedUsage perhaps, if we are going with that in UI. Why we did not want to use lastSeen is that in project list, what does it mean. When did the user last open project or what was last seen? So we thought of a very explicit term: lastReportedUsage, which seems very clear.

Reply:
Yeah, for sure. I'm happy to go with that. Though we might wanna be even more explicit, and call them something like `lastReportedFlagUsage` and `lastFlagUpdate` or something like that?